app/styles/_log.less

```diff
@@ -165,17 +166,6 @@
     }
   }
   &.log-fixed-height {
-    .ellipsis-loader {
```

Comment:
`log-fixed-height` is used on the monitoring page. Have you confirmed this doesn't break anything there?

Reply:
It doesn't. This should have been removed when`.ellipsis-loader` replaced by `.ellipsis-pulser`